classes/controller/AdminController.php

```diff
@@ -2269,8 +2269,11 @@ public function initFooter()
         //rtl.js overrides inline styles
         //iso_code.css overrides default fonts for every language (optional)
         if ($this->context->language->is_rtl) {
-            $this->addJS(_PS_JS_DIR_.'rtl.js');
-            $this->addCSS(__PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/css/'.$this->context->language->iso_code.'.css', 'all', false);
+            $this->addJS(_PS_JS_DIR_.'rtl.js', true);
+            $css = __PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/css/'.$this->context->language->iso_code.'.css';
+            if (file_exists($css)) {
```

Comment:
`is_string($css_file) && strlen($css_file) > 1`

Only if the variable is not empty, not if the file exist. 
And it's the only case (it's why this code is there) where your forced the links (if i'm good remember)